supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -16,27 +16,54 @@ When the format _day Month year_ causes a presentation or clarity issue, use _Mo
 The Conscious Language Group supports the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects with which Red Hat is involved.
 For more information about the Conscious Language Group, see https://github.com/conscious-lang/conscious-lang-docs.
 
+
 === blacklist and whitelist
 
-When possible, rewrite documentation to avoid these terms.
+When it is not possible to remove the terms _blacklist_ and _whitelist_, replace them with one of the following alternatives:
 
+* blocklist / allowlist: This combination is recommended by _The IBM Style Guide_. Use this combination unless your product area has another specific replacement that is agreed between engineering leadership and your documentation team.
+* denylist / allowlist
+* blocklist / passlist
 
-.Example: Removing 'blacklist'
-image:images/no.png[no] Heat blacklists any servers in the list from receiving updated heat deployments. After the stack operation completes, any blacklisted servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
+.Examples
+* Removing blacklist
 
-image:images/yes.png[yes] Heat excludes any servers in the list from receiving updated heat deployments. After the stack operation completes, any excluded servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
+image:images/no.png[no] Heat _blacklists_ any servers in the list from receiving updated heat deployments. After the stack operation completes, any blacklisted servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
 
+image:images/yes.png[yes] Heat _excludes_ any servers in the list from receiving updated heat deployments. After the stack operation completes, any excluded servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
 
-.Example: Removing 'whitelist'
-image:images/no.png[no] The following steps demonstrate adding a new rule to whitelist a custom binary.
+* Removing whitelist
 
-image:images/yes.png[yes] The following steps demonstrate adding a new rule to allow a custom binary.
+image:images/no.png[no] The following steps demonstrate adding a new rule to _whitelist_ a custom binary.
+
+image:images/yes.png[yes] The following steps demonstrate adding a new rule to _allow_ a custom binary.
+
+
+=== master and slave
+
+You can use the following alternatives for _master_ /_slave_:
+
+* primary / secondary
+* source / replica
+* initiator, requester / responder
+* controller, host / device, worker, proxy
+* director / performer
+
```

Comment:
I think that we also need to add the clause about or any other combination that is approved for your product. For example, on OpenShift, our replacement is control plane / worker (or compute). S

Reply:
+1